### DISCUSSION TITLE
Keeshare documentation

### DISCUSSION THREAD

**Discussion by JD-aus** [keepassxreboot/keepassxc#6682] [Q&A] [answered]
Hi All, love keepassxc, but the keeshare feature needs some more detailed documentation. [Keeshare Docs](https://keepassxc.org/docs/KeePassXC_UserGuide.html#_database_sharing_with_keeshare)

I have setup several share files and this works perfectly with my own KDB. now I want to get this working with multiple clients, but can't even get the second client to add the keeshare file without it creating a new one (if set to sync) or just displaying a blank group if setup for read only.

Thankfully i have backups, as i have learnt from previous failures with keepass dbs but i think i am getting stuck on importing the certificate for trusting the share file or something.

anyone have any documentation or notes or anything, if i can get this working i am happy to write a how-to about it so that others don't have to go through the pain that i have, but im not out of the woods yet.

**Comment by droidmonkey** [comment, marked as answer]
I am completely recoding keeshare because of the issues you have run into. It is not user friendly and results in very bad outcomes if not done exactly as the original designers intended. The current documentation is sufficient for simple sharing, but I don't encourage keeshare use in complicated scenarios.

**Reply by JD-aus** [reply]
If you need a beta tester or help with documentation (not sure if it's on GitHub or not), happy to help out. I'm not much of a coder, but I can hack my way around others code when the need arises.
